command/pbft_genesis/genesis.go

```diff
@@ -0,0 +1,180 @@
+package pbft_genesis
```

Comment:
Please follow the correct semantics, this command is to create the `polybft` genesis consensus protocol. `pbft` is the consensus engine.